backend/src/models/User.js

```diff
@@ -104,4 +104,8 @@ module.exports = {
     target: 'Location',
     direction: 'out',
   },
+  allowEmbedIframes: {
+    type: 'boolean',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-23T10:49:44Z" title="Monday, September 23rd 2019, 12:49:44 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -104,4 +104,8 @@ module.exports = {
     target: 'Location',
     direction: 'out',
   },
+  allowEmbedIframes: {
+    type: 'boolean',
```


You could even think about using a `date` here. The `false` would be if it's null.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-23T14:01:42Z" title="Monday, September 23rd 2019, 4:01:42 pm +02:00">Sep 23, 2019</time>_

---


I like that suggestion... maybe it's useful to know at some point when the user opted in to this "feature"

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-23T14:02:36Z" title="Monday, September 23rd 2019, 4:02:36 pm +02:00">Sep 23, 2019</time>_

---


Although, it would have to be the `String` representation of a `datetime`, no @roschaefer?